__tests__/components/forecast-summary.test.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ForecastSummary from '../../src/components/forecast-summary';
+
```

Comment:
This is another place where the `beforeEach()` would make everything a lot more DRY.